harvester/src/main/java/com/github/reflectoring/infiniboard/harvester/source/url/UrlSourceJob.java

```diff
@@ -1,15 +1,19 @@
 package com.github.reflectoring.infiniboard.harvester.source.url;
 
+import javax.net.ssl.SSLHandshakeException;
 import java.io.IOException;
+import java.net.UnknownHostException;
 import java.security.KeyManagementException;
 import java.security.KeyStoreException;
 import java.security.NoSuchAlgorithmException;
+import java.text.MessageFormat;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.text.MessageFormat'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)